src/utilities/utilities.ts

```diff
@@ -33,12 +68,16 @@ export async function execFile(
     executable: string,
     args: string[],
     options: cp.ExecFileOptions = {},
-    folderContext?: FolderContext
+    folderContext?: FolderContext,
+    customSwiftRuntime = true
 ): Promise<{ stdout: string; stderr: string }> {
     folderContext?.workspaceContext.outputChannel.logDiagnostic(
         `Exec: ${executable} ${args.join(" ")}`,
         folderContext.name
     );
+    if (configuration.runtimePath.length > 0 && customSwiftRuntime) {
```

Comment:
Do we need to check `configuration.runtimePath.length` here given the same check is in `swiftRuntimeEnv`